### DISCUSSION TITLE
[Feature] Multi select faces

### DISCUSSION THREAD

**Discussion by Daniel-PT** [immich-app/immich#8557] [Feature Request]
### The feature

I have alot of faces i dont need to see.
I could be nice to "multi-select" and remove.

Meny thanks for all the great work! Love it!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Biepa** [comment]
I think this is a duplicate of https://github.com/immich-app/immich/discussions/5142
